performance/README.md

```diff
@@ -0,0 +1,16 @@
+# Performance Regression Testing
+This directory includes dbt project setups to test on and a test runner written in Rust which runs specific dbt commands on each of the projects. Orchestration is done via the GitHub Action workflow in `/.github/workflows/performance.yml`. The workflow is scheduled to run every night, but it can also be triggered manually.
+
+The github workflow hardcodes our baseline branch for performance metrics as `0.20.latest`. As future versions become faster, this branch will be updated to hold us to those new standards.
```

Comment:
We'll need some way to keep this up to date. We already have `0.21.latest`, though there hasn't yet been a final release from that branch.

I'm wondering if it would be preferable to install the latest stable release of dbt, rather than checking out a specific branch?

Reply:
yeah we could do that instead. Definitely less manual than this approach.

Reply:
Maybe we should switch to this approach after a version or two so we know we haven't introduced any slippage in the metrics we're measuring. 🤔